<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Magna | Chimera HRE Control System</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: 1px black</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>421</width>
     <height>481</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid gray</string>
   </property>
   <property name="title">
    <string>CONTROL</string>
   </property>
   <widget class="QPushButton" name="button_control_estop">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>361</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>E-STOP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_control_auto">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>361</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>AUTO-START</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_control_fire">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>361</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(255, 255, 0);</string>
    </property>
    <property name="text">
     <string>FIRE</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_control_manual">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabs">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>40</y>
     <width>1301</width>
     <height>1021</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_hardware">
    <attribute name="title">
     <string>HARDWARE</string>
    </attribute>
    <widget class="QListView" name="hardware_listview_ports">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>301</width>
       <height>281</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="hardware_label_detectedDevices">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>DETECTED DEVICES</string>
     </property>
    </widget>
    <widget class="QListView" name="hardware_listview_portInfo">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>70</y>
       <width>431</width>
       <height>281</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
    <widget class="QListView" name="listview_hardware_serial_monitor">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>400</y>
       <width>471</width>
       <height>571</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkbox_hardware_autoscroll">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>374</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Autoscroll</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_graph">
    <attribute name="title">
     <string>LIVE GRAPH</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QProgressBar" name="bar_pressure">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>640</y>
     <width>81</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> QProgressBar::chunk {
     background-color: rgb(85, 170, 127);
 }</string>
   </property>
   <property name="value">
    <number>55</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="format">
    <string>%p psi</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_pressure">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>800</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>19</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px</string>
   </property>
   <property name="text">
    <string>PRESSURE</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="bar_temperature">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>640</y>
     <width>81</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> QProgressBar::chunk {
     background-color: rgb(85, 170, 127);
 }</string>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="format">
    <string>%pÂ°C</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_temperature">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>800</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>19</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px</string>
   </property>
   <property name="text">
    <string>TEMP.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_pressure">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>860</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>3</number>
   </property>
   <property name="value" stdset="0">
    <double>101.000000000000000</double>
   </property>
   <property name="intValue" stdset="0">
    <number>101</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_temperature">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>860</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>3</number>
   </property>
   <property name="value" stdset="0">
    <double>101.000000000000000</double>
   </property>
   <property name="intValue" stdset="0">
    <number>101</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_temperature_units">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>900</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>19</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px</string>
   </property>
   <property name="text">
    <string>Â°C</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_pressure_units">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>900</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>19</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px</string>
   </property>
   <property name="text">
    <string>PSI</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="bar_load">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>640</y>
     <width>81</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> QProgressBar::chunk {
     background-color: rgb(85, 170, 127);
 }</string>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="format">
    <string>%p N</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_load">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>800</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>19</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px</string>
   </property>
   <property name="text">
    <string>LOAD</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_load">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>860</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>3</number>
   </property>
   <property name="value" stdset="0">
    <double>101.000000000000000</double>
   </property>
   <property name="intValue" stdset="0">
    <number>101</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_temperature_units_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>900</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>19</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px</string>
   </property>
   <property name="text">
    <string>N</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
